lightly/api/api_workflow_upload_metadata.py

```diff
@@ -161,8 +162,9 @@ def upload_custom_metadata(self,
         }
 
         samples = retry(
-            self._samples_api.get_samples_by_dataset_id,
-            dataset_id=self.dataset_id
+            self._samples_api.get_samples_partial_by_dataset_id,
```

Comment:
is the sample id also returned here because it's needed two lines below?

Reply:
yes, id and filename is returned